.github/workflows/gradle.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         with:
           image: ${{ matrix.container }}
           options: -v ${{ github.workspace }}:/work -w /work -e ARTIFACTORY_PUBLISH_USERNAME -e ARTIFACTORY_PUBLISH_PASSWORD -e GITHUB_REF -e CI
-          run: df . && rm -f semicolon_delimited_script && echo $GITHUB_REF && ./gradlew build --build-cache -PbuildServer -PskipJavaFormat ${{ matrix.build-options }} ${{ env.EXTRA_GRADLE_ARGS }}
+          run: sudo apt-get update && sudo apt-get install -y gfortran && df . && rm -f semicolon_delimited_script && echo $GITHUB_REF && ./gradlew build --build-cache -PbuildServer -PskipJavaFormat ${{ matrix.build-options }} ${{ env.EXTRA_GRADLE_ARGS }}
```

Comment:
Where is this used in the build?  It's odd that it's only added in Gradle and not in the cmake builds?  If it's needed, the build system should be checking for it; this should go into the docker image (see the docker-images repo) and added to the README as a build dependency.

Reply:
A static libgfortran is needed because of the dependency on LAPACK/BLAS.

Reply:
Should it still be moved into the docker image?

Reply:
If it's required, it needs to be added to the docker image and the README.

Reply:
gfortran should be added as a build dependency in the docker-images README? or the wpical readme?

Reply:
It needs to be added to the docker image itself, and the allwpilib README's requirements section for Linux (so people know what is required to build allwpilib).